WHO Flu elderly vaccine coverage

spoonerf opened the pull request.

paarriagadap commented on etl/steps/data/meadow/who/2023-04-03/flu_elderly.meta.yml:
```diff
@@ -0,0 +1,13 @@
+      doses:
+        title: Number of doses
+        unit: doses
+        short_unit: ''
```
Maybe you can add here
```
        display:
          numDecimalPlaces: 0
```

paarriagadap commented on etl/steps/data/garden/who/2023-04-03/flu_elderly.py:
```diff
@@ -0,0 +1,54 @@
+    #
+    log.info("flu_elderly.harmonize_countries")
+    df = geo.harmonize_countries(
+        df=df, countries_file=paths.country_mapping_path, excluded_countries_file=paths.excluded_countries_path
```
Optional: you can also delete `excluded_countries_file=paths.excluded_countries_path` together with the `flu_elderly.excluded_countries.json` file and the steps would still run

paarriagadap commented on etl/steps/data/meadow/who/2023-04-03/flu_elderly.py:
```diff
@@ -0,0 +1,49 @@
+
+    # Load data from snapshot.
+    df = pd.read_excel(snap.path)
+    df = df[:-1]
```
Maybe you can comment on these steps to come back and not forget what you were doing

paarriagadap commented on etl/steps/data/meadow/who/2023-04-03/flu_elderly.py:
```diff
@@ -0,0 +1,49 @@
+
+    df = df.rename(columns={"NAME": "country", "YEAR": "year"}).set_index(["country", "year"], verify_integrity=True)
+
+    # df = df.set_index(["country", "year"])
```
You can delete this commented code, as you are creating an index just before

paarriagadap commented on snapshots/who/2023-04-03/flu_elderly.py:
```diff
@@ -0,0 +1,36 @@
+"""Script to create a snapshot of dataset 'Influenza Vaccination Coverage (WHO; UNICEF, 2023)'."""
```
Maybe some additional instructions on how to get the Excel file are needed, in case someone else updates this.

spoonerf replied:
Good point! Will add

paarriagadap commented on etl/steps/data/garden/who/2023-04-03/flu_elderly.py:
```diff
@@ -0,0 +1,54 @@
+"""Load a meadow dataset and create a garden dataset."""
```
Maybe a little more context for these steps can help (I would like `walkthrough` did this anyway)

paarriagadap commented on etl/steps/data/grapher/who/2023-04-03/flu_elderly.py:
```diff
@@ -0,0 +1,37 @@
+"""Load a garden dataset and create a grapher dataset."""
```
Maybe a little more context for these steps can help (I would like `walkthrough` did this anyway)

paarriagadap commented on etl/steps/data/meadow/who/2023-04-03/flu_elderly.py:
```diff
@@ -0,0 +1,49 @@
+"""Load a snapshot and create a meadow dataset."""
```
Maybe a little more context for these steps can help (I would like `walkthrough` did this anyway)

paarriagadap reviewed (commented):
Hi! Looks great! The code runs without issues. I added some minor comments; feel free to include them

paarriagadap commented:
The errors come from another place. Maybe @Marigold know what to do with them.

spoonerf commented:
> Hi! Looks great! The code runs without issues. I added some minor comments; feel free to include them

Thanks @paarriagadap! I think I've addressed all your comments now, will merge shortly.
